счёт строчных и прописных.py

```diff
@@ -0,0 +1,19 @@
+# !Это решение для варианта, когда нет знаков препинания!
+# Его легко можно модернизировать для вариата СО знаками препинания
+# Создадим по аналогии ещё 1 список со строчными буквами и посчитаем их
+
+userIn = input("Введите своё предложение\n")
+# уберём пробелы, чтобы не мешали считать
+userIn = userIn.replace(" ", "")
+# список из прописных англ. букв
+cap = list("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+countCap = 0
+# проверяем есть ли буква из списка и если есть, то считаем сколько раз есть
+for el in cap:
+    if el in userIn:
+        a = userIn.count(el)

```

Comment:
it's O(n**2) complexity. 
It can be solved with O(n) with one for and islower() method

Reply:
fixed